### ISSUE TITLE
Tag import doesn't work.

### ISSUE THREAD

**Issue by nimbus2332** [files-community/Files#13352]
### Description

I'm trying to import ready-made tag data from one PC to another, the files are in OneDrive, so the paths match 100%.

But, the second Pc does not import tag data, it seems to me that importing tags into a register (regedit) does not work. 

I also get an error: I run the application on behalf of the Admin → I'm trying to import an archive of settings.

### Steps To Reproduce

I'm trying to import ready-made tag data from one PC to another, the files are in OneDrive, so the paths match 100%.

But, the second Pc does not import tag data, it seems to me that importing tags into a register (regedit) does not work. 

I also get an error: I run the application on behalf of the Admin → I'm trying to import an archive of settings.

### Requirements

- e.g. Fix crash that occurs when...

### Files Version

2.5.21.0

### Windows Version

10.0.22621.2134

### Log File

[debug.log](https://github.com/files-community/Files/files/12577209/debug.log)

**Comment by nimbus2332** [comment]
I also tried to change the tag ID on the second PC, but it all doesn't work as if filetags.db was not imported.

I also turned on the display of alternative data and did not see the files I needed:

This is the first PC:

![1](https://github.com/files-community/Files/assets/54532528/8628fa77-ce8a-484a-ab1a-889658a1f8a9)

But on the second PC there is no such file.

**Comment by Josh65-2201** [comment]
Thanks for the report, OneDrive is known not keep ADS Files, see #12665. You can check Files settings > Tags to see if they imported but to move them you will need a NTFS drive to move them.

**Comment by nimbus2332** [comment]
> Thanks for the report, OneDrive is known that it doesn't keep ADS files, see #12665. You can check Files settings > Tags to see if they imported but to move them you will need a NTFS drive to move them.

Yes, I noticed that the hidden file in the screenshot is not uploaded to OneDrive.

The tags (name) were successfully imported, and even the ID in filetags.json matches the data from the first PC, but the database does not work. Apparently due to the lack of these - jpg:files

I also tried directly replacing filetags.db from the first PC to the second, but that didn't help either.

**Can this be fixed, and is it worth waiting for a fix?**
